### DISCUSSION TITLE
Beta Tag font is rendered unclearly

### DISCUSSION THREAD

**Discussion by mattxwang** [community/community#13140] [Other Feature Feedback, Questions, & Ideas]
When I select the "For You" feed, a bold style is applied to the "Beta" tag. It changes the font weight which makes the tag somewhat hard to read; compare 

![Screen Shot 2022-03-21 at 10 54 30 AM](https://user-images.githubusercontent.com/14893287/159334475-a184d960-d078-49db-acc0-8b258cce0581.png)

to 

![Screen Shot 2022-03-21 at 10 54 49 AM](https://user-images.githubusercontent.com/14893287/159334525-5adfa89d-a190-4eb6-81b8-e3e7d1e142a2.png)

For reference: I'm on:

* Chorme Version 99.0.4844.83
* macOS 12.3 (non-ARM)

However, I can also replicate this on Firefox:

![Screen Shot 2022-03-21 at 10 57 28 AM](https://user-images.githubusercontent.com/14893287/159335078-c60cfee6-161d-4685-8086-cdb3db4d7354.png)

Perhaps the tag shouldn't be bold, or otherwise the styling should be adjusted to make it render better!

**Comment by jfullstackdev** [comment]
this is what I'm seeing on Chrome,

![image](https://user-images.githubusercontent.com/47092464/182751106-1efc4aed-9178-4520-a0e8-13d42d5e7441.png)

also, 

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by mattxwang** [reply]
Ah, I had submitted this when the issue had just come out - agreed that it's fixed now. Thanks for checking in!
